osmaxx/excerptexport/forms/excerpt_form.py

```diff
@@ -30,30 +37,43 @@ def __init__(self, *args, **kwargs):
         self.helper.layout = Layout(
             Fieldset(
                 _('Excerpt'),
-                Field('name'), 'is_public',
+                Field('name'),
+                Field('bounding_geometry'),
+                'is_public',
             ),
-            BBoxBoundingGeometryMixin(self).form_layout(),
             OrderOptionsMixin(self).form_layout(),
             Submit('submit', 'Submit'),
         )
 
+    def clean(self):
+        clean = super().clean()
+        if 'bounding_geometry' in self._errors:
+            # need to add the hidden field validation error to a visible dom
+            self.add_error(None, _('Area Invalid or none selected.'))
+        return clean
+
+    def clean_bounding_geometry(self):
+        allowed_types = ['Polygon', 'Multipolygon']
+        data = self.cleaned_data['bounding_geometry']
+        data_dict = json.loads(data)
+        if data_dict.get('type') not in allowed_types:
+            raise ValidationError(_('Only {} are allowed as input.'.format(allowed_types)))
+        if data_dict.get('type') == 'Polygon':
+            data_dict['coordinates'] = [data_dict.get('coordinates')]
```

Comment:
only readability: I thought that 
`data_dict['coordinates'] = [data_dict.get('coordinates')]`
is more easy to read than
`data_dict['coordinates'] = [data_dict['coordinates']]`

(too many parenthesis).